### DISCUSSION TITLE
How to easily review captured videos

### DISCUSSION THREAD

**Discussion by Anonymouse83** [Motion-Project/motion#1713] [General]
I am a newbie to Motion, and have just (fingers crossed) got Motion to be stable.
I am running Ubuntu 20.04 on a reasonably powerful server.
I swapped from ZoneMinder as with a couple of cameras it was taking 100% CPU and corrupting the drive.
Now, with more cameras, it is not using 100% CPU - but ...
It accumulates videos, and I am struggling to find a way to easily review the videos as per ZoneMinder did.
I have extensively searched the Internet (but maybe not all of it!) for some help, and the only suggestions seem to be to go to ZoneMinder or one of the proprietary products, which are expensive for a domestic system.
Any help would be most appreciated.
Cheers!

**Comment by gabbymouth** [comment]
One easy way is to use nautilus or thunar to show preview thumbnails for video files with the files sorted by newest first.
